+++
widget = "blank"  # See https://sourcethemes.com/academic/docs/page-builder/
headless = true  # This file represents a page section.
active = true  # Activate this widget? true/false
title = "NeurIPS 2021 Competition"
+++

## Summary

The rapid advances in data-driven approaches to biology in recent
years have introduced new frontiers for research, exploration, and
understanding of biological processes at a resolution never before
seen. Cellular processes can now be characterized from measurements
collected at single cell resolution, thus capturing heterogeneity and
specificity without succumbing to the blurring effects of bulk
averaging over multiple cells in a sample. However, with the promising
potential for new discoveries, single cell data processing and
analysis has introduced a new set of challenges due to high
dimensionality, sparsity, nontrivial collection artifacts, batch
effects, and multiple modalities used to quantify cellular
measurements. These are further exacerbated by the destructive nature
of popular sequencing technologies, which destroy the observed cell as
part of the data collection process, thus not enabling repeated
sampling of the same cell. This proposal is based on our conjecture
that recent advances in representation learning (in particular, in
fields involving self supervised, multi-view, and transfer learning)
hold the key to alleviating such challenges to lift some of the main
barriers in the field and unlock new capabilities bridging data
science, machine learning, and computational biology. To this end, we
propose to hold a multi-modal single cell data analysis competition to
expose the machine learning community to predominant problems
encountered in biomedical multiomic applications, and the
computational biology community to a wider range of state of the art
machine learning methods, which may not be traditionally associated or
applied to biomedical data. We ask participants to learn meaningful
joint embeddings from multimodal data and to infer missing or
corrupted features. We also include a track for generating synthetic
multimodal single cell data realistic data and noise distributions to
enable more rigorous benchmarking in the field.

## Details

Stay tuned for more information!

## Organizing Team

All organizers are listed in alphabetical order.

-   **Daniel Burkhardt** (Platform Administrator) is a machine learning
    scientist at Cellarity. He is a core organizer of the Open Problems
    in Single-Cell Analysis project. He completed his PhD in Genetics at
    Yale University with a specialization in machine learning under the
    supervision of Smita Krishnaswamy. His dissertation focused on
    modelling experimental perturbations and biological heterogeneity in
    single-cell datasets. Daniel is also the core organizer of the
    Machine Learning for Single Cell Analysis workshop, offered
    bi-annually since 2019 by the Krishnaswamy Lab at Yale.

-   **Smita Krishnaswamy** is an associate professor of Computer Science and
    Genetics at Yale University. Her research focuses on unsupervised
    learning, using data geometry, topology and deep learning methods
    for big biomedical data. She has extensive experience organizing
    workshops including the Learning Meaningful Representations of Life
    Workshop at NeurIPS2020, Topological Data Analysis workshop at
    NeurIPS 2020, and Geometric and Topological Representation Learning
    at ICLR2021. The Krishnaswamy Lab also offers bi-annual workshops in
    Machine learning for Single Cell Analysis
    [krishnaswamylab.org/workshop](krishnaswamylab.org/workshop), with
    Dr. Krishnaswamy being the main instructor. Dr. Krishnaswamy also
    co-organized the Open Problems in Single Cell Analysis Jamboree in
    March 2021, which sets the framing for this proposal. Smita will be
    responsible for overseeing the coordinating the event, overall
    design of tasks and metrics, facilitating connections with sponsors,
    as well as other workshops at NeurIPS.

-   **Malte Lücken** is a senior postdoctoral researcher and team leader
    for integrative single-cell analysis in the Machine Learning Group
    of Prof. Fabian Theis. His research focuses on evaluating
    computational methods for single-cell analysis and investigating how
    environmental stimuli and natural variation manifests on the level
    of single cells. He has experience organizing hackathons including
    for the Single Cell Omics Germany network, Helmholtz Munich, and the
    Open Problems project, which provides the context for this proposal.
    He is further a co-organizer of the "Advances and Challenges in
    Single Cell and Spatial Data Analysis\" workshop for ISMB 2021.
    Malte will be responsible for administrating the platform and
    providing baseline methods for comparison.

-   **Debora Marks** (Liason to LMRL and larger biological community) is an
    Associate Professor in the Department of Systems Biology at Harvard
    Medical School. As a mathematician and computational biologist,
    Marks develops novel algorithms, statistical methods and machine
    learning to successfully address unsolved biological problems. Her
    passion is to understand, predict and design biomolecular systems in
    a way that impacts biomedical applications and synthetic biology at
    many scales, with a focus on developing new methods in probabilistic
    modeling that exploit the huge and increasing corpus of natural and
    synthetic sequence diversity. Over the past five years, her lab has
    developed methods that accelerate structural biology with
    applications to cryoEM, crystallography, protein design and computed
    3D structures of thousands of proteins with unknown folds, protein
    complexes and RNA interactions, as well as flexible, dynamic and
    even disordered protein ensembles. To address new challenges in
    protein design, Marks has adapted and developed NLP-inspired deep
    neural methods for (i) designing libraries of high affinity,
    specific nanobodies, antibodies that bypass the need for expensive
    rounds of selection or labor intensive specificity assays and (ii)
    design and prediction of proteins that induce membranes
    compartmentalization and potentially biostasis in human cells
    and (iii) predicting the effect of genetic variation on disease and
    drug response. In recent years she has been organizing multiple
    events creating a bridge between computational biology and machine
    learning, including the LMRL (learning meaningful representation of
    life) workshop series and the recurring ICML workshop on
    computational biology.

-   **Bastian Rieck** is a senior assistant in the Machine Learning and
    Computational Biology Lab of Prof. Dr. Karsten Borgwardt at ETH
    Zurich. His main research interests are manifold learning approaches
    based on concepts from topological data analysis, with a focus on
    personalized medicine and computational biology. Bastian will assist
    in the evaluation of all methods and support the construction of an
    automated testing/evaluation platform. He organized a workshop on
    "Topological Data Analysis and Beyond" at NeurIPS 2020, as well as a
    workshop on "Geometrical and Topological Representation Learning" at
    ICLR 2021. He was also involved in organizing a track on Clinical
    Precision Medicine for the Applied Machine Learning Days 2021.

-   **Jian Tang** is currently an assistant professor at HEC Montreal
    (business school of University of Montreal) and a core faculty
    member at Mila-Quebec AI Institute, which is a research institute
    focusing on deep learning and reinforcement learning led by Turing
    Award Winner Yoshua Bengio. His research focuses on graph
    representation learning, graph neural networks, drug discovery, and
    knowledge graphs. He is named to the first cohort of Canada CIFAR
    Artificial Intelligence Chairs (CIFAR AI Research Chair). He was a
    research fellow in University of Michigan and Carnegie Mellon
    University. He was a researcher in Microsoft Research Asia for two
    years. He received the best paper award of ICML'14 and nominated for
    the best paper of WWW'16. He co-organized a tutorial on graph
    representation learning at KDD 2017, AAAI 2019, and AAAI 2021, and
    co-organized a few workshops on graph representation learning at SDM
    2019, CIKM 2019, AAAI 2020, and ICML 2020.

-   **Fabian Theis** is the Professor of Mathematical Modeling of Biological
    Systems at the Department of Mathematics of the Technical University
    of Munich; the director of the Institute of Computational Biology,
    Helmholtz Zentrum München; the scientific director of the Helmholtz
    Artificial Intelligence Cooperation Unit; and a faculty member of
    the Wellcome Trust Sanger Institute, Cambridge, UK. Furthermore, he
    is part of the scientific advisory boards of the Berlin Institute of
    Health, the European Molecular Biology Laboratory (EMBL), as well as
    a member of the steering committee of the Human Cell Atlas Lung Seed
    Network, the Analysis Working Group of the Human Cell Atlas; and the
    Data Pipeline Committee of the Human Cell Atlas Initiative amongst
    others. Dr. Theis has been part of the organizing team of several
    national and international conferences. Most recently these include
    the Single Cell Biology virtual conference 2020 and the Helmholtz AI
    founding working 2020 (150 participants). He has also chaired and
    co-chaired workshops within Germany such as the 'Computational
    Single Cell Genomics' workshop in Munich ( 150 attendees), and
    international conferences (e.g., the 'International Conference on
    Artificial Neural Networks'; ICANN 2019). (data provider)

-   **Alexander Tong** is a PhD candidate in Computer Science in the lab of
    Prof. Smita Krishnaswamy at Yale University. His research focuses on
    manifold learning and optimal transport with a focus on single-cell
    data. Alexander will be responsible for supporting the comparison
    infrastructure and evaluating submissions.

-   **Guy Wolf** is an assistant professor in the Department of Mathematics
    and Statistics at the Université de Montréal and a Core Member of
    Mila---the Québec AI Institute. He is also affiliated with
    IVADO---the institute of data valorization at Montreal. His research
    focuses on manifold learning, representation learning, and geometric
    deep learning for exploratory data analysis, including methods for
    dimensionality reduction, visualization, denoising, data
    augmentation, and coarse graining, with particular focus on
    applications in biomedical data exploration. His recent organization
    activities include the CRM-IVADO Industrial Problem Solving Workshop
    (IPSW 2020), a two-part mini-symposium on geometric deep learning as
    part of the inaugural SIAM Conference on Mathematics of Data Science
    held online in June 2020, a NeurIPS 2020 workshop on topological
    data analysis, an ICLR 2021 workshop on geometric and topological
    representation learning, and a BIRS workshop on geometric deep
    learning.
